doc/source/faq/vespa-ai-integration.rst

```diff
@@ -0,0 +1,79 @@
+.. _faq-vespa-ai-integration:
+
+.. meta::
+   :description: Configure rsyslog to forward events to Vespa.ai using omhttp and Vespa's Document v1 API.
+   :keywords: vespa, omhttp, document api, search, analytics
+
+.. summary-start
+
+Learn how to push rsyslog events into Vespa.ai's search platform with omhttp and Vespa's Document v1 API.
+
+.. summary-end
+
+FAQ: Sending Logs from rsyslog to Vespa.ai
+==========================================
+
+`Vespa.ai <https://vespa.ai>`_ is an open-source engine for real-time search, recommendation, and analytics at scale. It ingests structured documents,
+indexes them for low-latency search, and executes machine-learned ranking or aggregations directly where the data lives.
+Connecting rsyslog to Vespa.ai lets you ship log events into a streaming analytics platform that can keep indexes and features
```

Comment:
I would change "logs" to "data" here and I would remove "streaming". While there's nothing wrong with sending logs to Vespa, and Vespa offers streaming functionality, I wouldn't call it a logging or a streaming platform.

But the rest is true, in the sense that Vespa can execute complicated ranking functions on top of massive amounts of data.

Reply:
yeah, you are of course right. That's also our track, but I have to admit that to handle fully generic data, some knobs need to be turned, and in order to make it really easy, some new knobs need to be added. But in general, we can extract and transform everything. Pure JSON of course works pretty well.